packages/client/src/app.js

```diff
@@ -54,13 +55,16 @@ app.use((state, emitter) => {
 
   state._session = new Proxy({}, {
     set: function(target, key, value) {
-      if (key === 'id') {
+      const ret = Reflect.set(target, key, value)
```

Comment:
Woah, what does this do? ...returns a boolean if `target[key] = value` was successful...? wat?

Conclusion: Proxy and Reflect are both magic. I realise this existed before but never noticed it's usage. thisandagain pls explain.